recipes/binutils/all/test_package/conanfile.py

```diff
@@ -1,22 +1,44 @@
-from conans import ConanFile, tools
+from conan import ConanFile, tools
 from io import StringIO
 import os
+import json
+import sys
 
 
 class TestPackageConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
+    test_type = "explicit"
+
+    def build_requirements(self):
+        self.build_requires(self.tested_reference_str)
```

Comment:
```suggestion
        self.tool_requires(self.tested_reference_str)
```
Both Conan v1 and v2 should use tool_requires, which is a replacement for build_requires: https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#requirements